romochka_bembot.js

```diff
@@ -56,11 +59,12 @@ const generateBemhtmlLink = bemhtml => {
  * @returns {String} Link
  */
 const generateBemjsonLink = bemjson => {
-	const trimedBemjson = _.trimEnd(bemjson, ';');
+	assert(typeof bemjson === 'string', 'Bemjson must be a string');
+
+	const trimedBemjson = bemjson.replace(/;+$/, '');
```

Comment:
@maksugr of course u shoud add 
`bemjson[bemjson.length - 1] === ';' && (bemjson = bemjson.trim().splice(0, -1))`
but, if you want it, ok) i'll make change back to lodash.
But maybe smth like [this](https://www.npmjs.com/package/lodash.trimend)? ^_^